### DISCUSSION TITLE
Per Class accuracy and visualization

### DISCUSSION THREAD

**Discussion by NaivePro1998** [rwightman/efficientdet-pytorch#200] [Q&A]
Hi,

Do anyone have code for individual class accuracy and also visualization. 

Appologies, If this is not something big, I am new to this field.

Regards,

**Comment by NaivePro1998** [comment]
Is there any such thing exist?

**Reply by rwightman** [reply]
It exists in the google evaluator I ported for using w/ pascal or open-images, but you give up the multi-point mAP calculations from the coco evaluator and just get @ 0.5 as a default. You have to customize the coco evaluator to get per class w/ multi-point mAP, you can google it as others have done it before.
